packages/react-app/src/hooks/ExchangePrice.js

```diff
@@ -22,7 +22,7 @@ export default function useExchangePrice(targetNetwork, mainnetProvider, pollTim
         setPrice(parseFloat(route.midPrice.toSignificant(6)));
       }
     }
-    getPrice();
+    // getPrice();
```

Comment:
Had to comment this. It throws this error:

![image](https://user-images.githubusercontent.com/2486142/128640442-7656b0fd-a495-4937-b357-0602fb40de23.png)